README.md

```diff
@@ -2,7 +2,7 @@
 
 # OpenMRS Android Client User Guide
 
-This is OpenMRS Android Client User Guide which will introduce any new user to the Android Client as well as serve as a guide for existing users. The guide will have detailed description of all the features along with some simple use cases. It will also contain FAQs, Troubshooting, Glossary, etc.
```

Comment:
Just to give an example, what is wrong with? "any new user"

Reply:
I chose 'new users' as I'd wanted it to be more direct, as 'new users' already represents every type of a new user, including 'any'. Also we are expecting the plural 'users' instead of 'user' in the sentence; e.g. "John didn't add any new users to his group chat". Using 'any new user' in that ex. sentence won't make sense since 'users' had to be plural.

Should it be **any new users** instead?

Reply:
@dkayiwa, I think "new users" is more appropriate.

Reply:
@f4ww4z, can you please resolve merge conflict so we can merge your PR?
Thanks for the work :)